src/calculateRentalCost.js

```diff
@@ -27,8 +27,24 @@
  * @return {number}
  */
 
-function calculateRentalCost(days) {
-  // write code here
-}
+function calculateRentalCost(days = 0) {
+  const sevenDays = 7;
+  const threeDays = 3;
+  const pricePerDay = 40;
+  const discFromSevenDays = 50;
+  const discFromThreeDays = 20;
+
+  if (days < threeDays) {
+    return days * pricePerDay;
+  };
+
+  if (days >= threeDays && days < sevenDays) {
```

Comment:
```suggestion
  if (days < sevenDays) {
```
ты уже проверил это в предыдущем if